aten/src/ATen/native/transformers/cuda/sdp_utils.cpp

```diff
@@ -176,11 +177,42 @@ bool check_sm_version(cudaDeviceProp * dprops) {
   return is_gte_lower_bound && is_lte_upper_bound;
 }
 
+#if USE_ROCM
+c10::once_flag gcn_arch_override_flag;
+const char* over_arch = nullptr;
```

Comment:
This introduces a global symbol `over_arch` sticking for everyone to see, which is wrong, but as it's limited to ROCM wheels this sounds fine to me.

Reply:
Oh this variable is actually inside an anonymous namespace nested inside namespace sdp.